### PR TITLE
Introduce SIGNALFX_SPAN_TAGS env var for global tags

### PR DESCRIPTION
**Pull request by johnbley**
- Introduce SIGNALFX_SPAN_TAGS with format `key1:val1,key2:val2`
- Add unit test for it
- Add doc line for it

### CODE REVIEW

File: src/tracer.js [hunk 1]
```diff
@@ -25,6 +25,22 @@ class SignalFxTracer extends Tracer {
 
     this._scopeManager = new ScopeManager()
     this._scope = new Scope()
+
+    this._extraTags = {}
+    if (process.env.SIGNALFX_SPAN_TAGS) {
+      for (const segment of process.env.SIGNALFX_SPAN_TAGS.split(',')) {
+        const kv = segment.split(':')
+        if (kv.length === 2 && kv[0].length !== 0 && kv[1].length !== 0) {
```
Comment (rmfitzpatrick) [inline src/tracer.js:33]: When this was done in go-tracing the key and value were trimmed to prevent undesired whitespace.

Comment (johnbley) [reply src/tracer.js:33]: Good call; will fix.

File: src/tracer.js [hunk 2]
```diff
@@ -25,6 +25,22 @@ class SignalFxTracer extends Tracer {
 
     this._scopeManager = new ScopeManager()
     this._scope = new Scope()
+
+    this._extraTags = {}
```
Comment (rmfitzpatrick) [inline src/tracer.js:29]: I think this logic should be in Config: https://github.com/signalfx/signalfx-nodejs-tracing/blob/master/src/config.js#L62

File: docs/API.md [hunk 3]
```diff
@@ -822,6 +822,7 @@ Options can be configured as a parameter to the `init()` method or as environmen
 | flushInterval |                              | 2000      | Interval in milliseconds at which the tracer will submit traces to the agent. |
 | experimental  |                              | {}        | Experimental features can be enabled all at once using boolean `true` or individually using key/value pairs. There are currently no experimental features available. |
 | plugins       |                              | true      | Whether or not to enable automatic instrumentation of external libraries using the built-in plugins. |
+|               | SIGNALFX_SPAN_TAGS           |           | Global tags for all spans (like config `tags`) in format `key1:val1,key2:val2`. |
```
Comment (rmfitzpatrick) [inline docs/API.md:825]: This can be provided in an updated line 820

**Pull request merged**